### DISCUSSION TITLE
[BETA RELEASE] R.M.S. TITANIC codbo4 Demake

### DISCUSSION THREAD

**Discussion by Cybrock9000** [nzp-team/nzportable#790] [Map Releases]
THIS IS 85% COMPLETE!
You are the last person alive on the Titanic. You and your fellas (if you do multiplayer) will be fighting endless hoards of zombies and unraveling the secrets within. This map is a demake of the Voyage of Despair map on codbo4. This is also my first map and knowing me (which you dont) i go big on projects. Everything in this map is made by me (like textures).
![Screenshot 2024-06-19 234427](https://github.com/nzp-team/nzportable/assets/118949998/94bc867a-ce1d-4904-ac7e-c04afe9e4974)
![Screenshot 2024-06-19 234509](https://github.com/nzp-team/nzportable/assets/118949998/ad7235c0-6ffb-4638-900b-4b94d53b0ef4)
![Screenshot 2024-06-19 234746](https://github.com/nzp-team/nzportable/assets/118949998/9389e44b-5587-435e-bf9c-f28a4a980cfe)
![Screenshot 2024-06-19 235347](https://github.com/nzp-team/nzportable/assets/118949998/354c84fc-ad07-4a80-bfab-c8b09feb7085)

Beta map download: https://github.com/Cybrock9000/M.R.S.-Titanic-Nzp-Map

**Comment by Cybrock9000** [comment]
Ive adjusted the download link to fit the "Map Release Standardization"

**Comment by Cybrock9000** [comment]
the pictures are out dated, but when it is fully finished i will make another release

**Comment by zqpvr** [comment]
hope you can get it done soon.

**Reply by Cybrock9000** [reply]
will probably be done tomorrow

**Reply by Cybrock9000** [reply]
or the next day (i just realized i need to do waypoints)

**Comment by ToonLink0001** [comment]
Yo this is cool, thanks for your contribution!

**Reply by Cybrock9000** [reply]
thanks hope you enjoy it :)

**Comment by Cybrock9000** [comment]
this post has been moved to https://github.com/nzp-team/nzportable/discussions/810
